### PR TITLE
Move lifx to own component

### PR DESCRIPTION
**Pull request by amelchio**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17201

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
We should add a page for the lifx light platform, and direct users to the component page.

File: source/_components/lifx.markdown [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+description: "Instructions on how to integrate LIFX into Home Assistant."
+date: 2016-01-27 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lifx.png
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.12
```
Comment (MartinHjelmare) [inline source/_components/lifx.markdown]: Update this, since the component is new.

Comment (amelchio) [reply source/_components/lifx.markdown]: That seems odd but okay. I am not sure what to put into `ha_category` for the component, though.

File: source/_components/lifx.markdown [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+description: "Instructions on how to integrate LIFX into Home Assistant."
+date: 2018-10-08 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lifx.png
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.80
```
Comment (MartinHjelmare) [inline source/_components/lifx.markdown]: Is it part of 0.80 beta?

Comment (amelchio) [reply source/_components/lifx.markdown]: Of course not, thank you :)

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**